WordPress/Classes/ViewRelated/Post/PostCardStatusViewModel.swift

```diff
@@ -63,8 +63,8 @@ class PostCardStatusViewModel: NSObject {
             return generateFailedStatusMessage()
         } else if post.remoteStatus == .pushing {
             return NSLocalizedString("Uploading post...", comment: "Message displayed on a post's card when the post has failed to upload")
-        } else if !post.hasLocalChanges() && post.hasAutosaveRevision {
-            return StatusMessages.hasUnsavedChanges
+        } else if post.hasVersionConflict || !post.hasLocalChanges() && post.hasAutosaveRevision {
```

Comment:
Now we're treating local and remote autosaves differently the changes in this hunk will likely be reverted.